package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "azure-devops-templates",
+  "version": "1.0.0",
+  "description": "templates to for Azure Devops build pipelines used by the hearings reforms team",
+  "main": "index.js",
+  "dependencies": {},
+  "devDependencies": {
+    "yaml-lint": "^1.2.4"
```

Comment:
nice

Reply:
can you approve this PR so i can verify my upcoming changes won't break the template please? Cheers